pipelines/environments/componentsdk-1.5-ubuntu20.04-py38-cpu/asset.yaml

```diff
@@ -0,0 +1,4 @@
+name: componentsdk-1.5-ubuntu20.04-py38-cpu
+type: environment
+spec: spec.yaml
+extra_config: environment.yaml
```

Comment:
Please add newlines at the end of your files.

Reply:
Added for all files in my branch.